nnpdf_data/nnpdf_data/commondata/ATLAS_WPWM_7TEV_46FB/filter_utils.py

```diff
@@ -0,0 +1,116 @@
+"""
+This module contains helper functions that are used to extract the data values 
+from the rawdata files.
+"""
+
+import yaml
+import pandas as pd
+import numpy as np
+
+
+def get_data_values():
+    """
+    returns the central data values in the form of a list.
+    """
+
+    data_central = []
+
+    hepdata_table_1 = f"rawdata/HEPData-ins1502620-v1-Table_9.yaml"
+    hepdata_table_2 = f"rawdata/HEPData-ins1502620-v1-Table_10.yaml"
+
+    with open(hepdata_table_1, 'r') as file:
+        input_1 = yaml.safe_load(file)
+
+    with open(hepdata_table_2, 'r') as file:
+        input_2 = yaml.safe_load(file)
+
+    values_1 = input_1['dependent_variables'][0]['values']
+    values_2 = input_2['dependent_variables'][0]['values']
+
+    values = values_1 + values_2
+
+    for value in values:
+        # store data central and convert the units
+        data_central.append(value['value'] * 1000)
+
+    return data_central
+
+
+def get_kinematics():
+    """
+    returns the kinematics in the form of a list of dictionaries.
+    """
+    kin = []
+
+    hepdata_table_1 = f"rawdata/HEPData-ins1502620-v1-Table_9.yaml"
+    hepdata_table_2 = f"rawdata/HEPData-ins1502620-v1-Table_10.yaml"
+
+    with open(hepdata_table_1, 'r') as file:
+        input_1 = yaml.safe_load(file)
+
+    with open(hepdata_table_2, 'r') as file:
+        input_2 = yaml.safe_load(file)
+
+    for i, M in enumerate(input_1["independent_variables"][0]['values']):
+
+        kin_value = {
+            'abs_eta': {
+                'min': None,
+                'mid': (0.5 * (M['low'] + M['high'])),
+                'max': None,
+            },  # absolute lepton eta
+            'm_W2': {'min': None, 'mid': 6463.838404, 'max': None},
+            'sqrts': {'min': None, 'mid': 7000.0, 'max': None},
+        }
+
+        kin.append(kin_value)
+
+    for i, M in enumerate(input_2["independent_variables"][0]['values']):
+
+        kin_value = {
+            'abs_eta': {
+                'min': None,
+                'mid': (0.5 * (M['low'] + M['high'])),
+                'max': None,
+            },  # absolute lepton eta
+            'm_W2': {'min': None, 'mid': 6463.838404, 'max': None},
+            'sqrts': {'min': None, 'mid': 7000.0, 'max': None},
+        }
+
+        kin.append(kin_value)
+
+    return kin
+
+
+def get_systematics_dataframe():
+    """
+    returns the absolute systematic uncertainties in the form of a pandas dataframe.
+    """
+    sys_rawdata_path = "rawdata/wzrap11_full.csv"
+
+    df = pd.read_csv(sys_rawdata_path)
```

Comment:
The reason for the UNCORR.1 error is this. If you read a csv file with repeated keys (like in this case) they just get an extra suffix, in this case `.1` because there are two UNCORR errors.

I guess at some point (perhaps when you are writing the numbers down) you should check whether the type startswith UNCORR and then write just that.